website/docs/en/config/experiments.mdx

```diff
@@ -334,3 +334,150 @@ module.exports = {
   },
 };
 ```
+
+## experiments.cache
+
+<ApiMeta addedVersion="1.2.0" />
+
+- **Type:** `ExperimentCacheOptions`
+
+- **Default:** production mode is `false`, development mode is `true`
+
+```ts
+type ExperimentCacheOptions =
+  | boolean
+  | {
+      type: 'memory';
+    }
+  | {
+      type: 'persistent';
+      buildDependencies?: string[];
+      version?: string;
+      snapshot?: {
+        immutablePaths?: Array<string | RegExp>;
+        unmanagedPaths?: Array<string | RegExp>;
+        managedPaths?: Array<string | RegExp>;
+      };
+      storage?: {
+        type: 'filesystem';
+        directory?: string;
+      };
+    };
+```
+
+Control experimental caching behavior. This will only work if [global cache](/config/cache) is set to `true`.
+
+### Disable cache
+
+Configuring `experiment.cache` to `false` to disable cache, which is no different from configuring the [global cache](/config/cache) to `false`.
```

Comment:
The Info in `/config/cache` can be updated now:

![image](https://github.com/user-attachments/assets/07f2efe0-bd6f-4e4e-8fe0-cecffa58c0ed)